client/routes.js

```diff
@@ -37,7 +37,11 @@ class Routes extends Component {
         <Route exact path="/products/faceshields" component={AllFaceshields} />
         <Route exact path="/products/sanitizers" component={AllSanitizers} />
         <Route exact path="/users" component={AllUsers} />
-        <Route exact path="/users/:userid" component={SingleUser} />
+        <Route
```

Comment:
I don't think we will need this change when we merge my change to the routes so we can just go back to the original line 40  `<Route exact path="/users/:userid" component={SingleUser} />`

Reply:
I think it might be good to change all userId to just id to avoid confusion but not necessary in this situation